### PR TITLE
Fix bundle. experimentor, rnd machines, mob throwing

### LINKED ISSUE #17513: experimentor machine has no sprite when its panel is open
[Note]: # (State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

[Admins]: # (If you are reporting a bug that occured AFTER you used varedit/admin buttons to alter an object out of normal operating conditions, please verify that you can re-create the bug without the varedit usage/admin buttons before reporting the issue.)

### LINKED ISSUE #17538: Using a screwdriver on the Circuit Imprinter makes the Protolathe not wor
I can still access the Circuit Imprinter menu however despite it being open. Resyncing the devices doesn't let me use the protolathe unless I reset R&D after opening the panels on every device while the monitor is unscrewed.

EDIT: The same thing happens with the Analyzer. I can't use the Analyzer despite it being synced to the RD computer howerver. Unscrewing doesn't show either the Imprinter of Analyzer unsyncing

### LINKED ISSUE #17609: Can't throw people around
~~#17355 was test-merged at the time(no clue if it affects anything).~~ not caused by this
No matter the grab type, you simply can't throw people around.

### PR DESCRIPTION
**Pull request by phil235**
Fixes panel opening deconnecting the wrong rnd console. Fixes #17538
Fixes not being able to throw mobs. Fixes #17609
Fixes experimentor having no sprite when panel open. Fixes #17513

All three bugs I caused.

### CODE REVIEW

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -208,42 +208,37 @@
...
 
-				add_logs(src, M, "thrown", addition="from [start_T_descriptor] with the target [end_T_descriptor]")
-
-	if(!item) return //Grab processing has a chance of returning null
+				add_logs(src, throwable_mob, "thrown", addition="from [start_T_descriptor] with the target [end_T_descriptor]")
 
-	if(!ismob(item)) //Honk mobs don't have a dropped() proc honk
-		unEquip(item)
-	if(src.client)
-		src.client.screen -= item
```
Comment (phil235) [inline code/modules/mob/living/carbon/carbon.dm]: 1: we should only be calling this for non mobs.
2: all this is already done in unEquip() now.

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -208,42 +208,37 @@
...
-
-	//actually throw it!
-	if(item)
-		item.layer = initial(item.layer)
```
Comment (phil235) [inline code/modules/mob/living/carbon/carbon.dm]: should only be called for non mobs, and it's already done by unEquip() now.

File: code/modules/mob/living/carbon/carbon.dm [hunk 3]
```diff
@@ -208,42 +208,37 @@
 	throw_mode_off()
 	if(!target || !isturf(loc))
 		return
-	if(istype(target, /obj/screen)) return
+	if(istype(target, /obj/screen))
+		return
 
-	var/atom/movable/item = src.get_active_hand()
```
Comment (phil235) [inline code/modules/mob/living/carbon/carbon.dm]: calling an atom movable "item" .... GENIUS!

**Comment by LatD** [comment]
I noticed now that experimentor had `linked_console.linked_destroy = null` too. Can you add that proc to it as well here?

**Comment by phil235** [comment]
@LatD That was a bug the experimentor is never equal to linked_console.linked_destroy. If we keep that line then screwdriving the experimentor will disconnect the potential destructive analyzer linked to the console.

**Comment by LatD** [comment]
I thought it was meant that only experimentor or DA gets connected and that was to prevent them both getting connected since experimentor can get the levels same way as DA. But if there's no downside to this then it's okay. Not like someone would use experimentor, honk.

**Comment by LatD** [comment]
Another related thing to this is the old bug in which you have to use a screwdriver to all 3 machines before the console accepts them for syncing. That's because the 3 machines retain their `linked_console` even when the main console is disconnected.

If you add
```
/obj/machinery/computer/rdconsole/disconnect_console()
	linked_lathe.linked_console = null
	linked_imprinter.linked_console = null
	linked_destroy.linked_console = null
	..()
```
to rdconsole.dm this should be solved.

**Comment by phil235** [comment]
will do

**Pull request merged**